.github/workflows/audit.yaml

```diff
@@ -5,7 +5,7 @@
 
 jobs:
   audit:
-    if: ${{ github.repository_owner == 'pantsbuild' }}
+    if: github.repository_owner == 'pantsbuild'
```

Comment:
The ${{ }} are unnecessary in an `if` - GHA always evaluates the contents of `if` as an expression - and removing them makes it easier to add other conditions later by string manipulation.